include/cell.h

```diff
@@ -222,6 +224,31 @@ class Cell {
                                     unsigned phase, double pvolume,
                                     const Eigen::Matrix<double, 6, 1>& pstress);
 
+  //! Assign velocity constraint
+  //! Directions can take values between 0 and Dim-1
+  //! \param[in] face_id Face of cell of velocity constraint
+  //! \param[in] dir Direction of velocity constraint
+  //! \param[in] velocity Applied velocity constraint
+  bool assign_velocity_constraint(unsigned face_id, unsigned dir,
+                                  double velocity);
+
+  //! Apply velocity constraints
+  void apply_velocity_constraints();
+
+  //! Compute normal vector
+  void compute_normals();
+
+  //! Return number of normal vectors
+  //! \retval number of normal vectors
+  unsigned nnormal() const { return face_normals_.size(); }
```

Comment:
Do we still need this function?

Reply:
I rather still have it to get the number of normals rather than specify 4 or 6 everytime

Reply:
It's used only in the cell, you can call `face_normals_.size()`, without having to define a public function?

Reply:
Agree, and we have `element/nfaces()`, thus this is deprecated

Reply:
Actually, I would still use it to check the number of faces. Or is it sufficient to know that whatever the element gives is right?

Reply:
`Element` should give it right, you have no other way of checking it in the cell.

Reply:
Understood